scripts/utils.js

```diff
@@ -0,0 +1,9 @@
+import { existsSync, copyFileSync } from 'fs';
+
+const getCommonJsTypesPath = dtsPath => dtsPath.replace(/.ts$/, '.cts');
```

Comment:
```suggestion
const getCommonJsTypesPath = dtsPath => dtsPath.replace(/\.ts$/, '.cts');
```

In your version a dot means any symbol